metricshub-web/react/src/components/config/Tree/ConfigTree.jsx

```diff
@@ -15,27 +23,101 @@ const ROOT_ID = "__config_root__";
  * @returns {JSX.Element} The configuration tree component.
  */
 export default function ConfigTree({ files, selectedName, onSelect, onRename, onDelete }) {
+	const dispatch = useAppDispatch();
+	const list = useAppSelector((s) => s.config.list);
 	const selectedIds = React.useMemo(() => (selectedName ? [selectedName] : []), [selectedName]);
 
+	// config root kebab
+	const [rootMenuAnchor, setRootMenuAnchor] = React.useState(null);
+	const openRootMenu = (e) => {
```

Comment:
Same remark here, handlers like openRootMenu, closeRootMenu and handleBackupAll are recreated every render,  use `useCallback`.